pkg/apis/solr/v1beta1/zz_generated.deepcopy.go

```diff
@@ -181,6 +181,28 @@ func (in *PersistenceSource) DeepCopy() *PersistenceSource {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *PodPolicy) DeepCopyInto(out *PodPolicy) {
+	*out = *in
+	if in.Affinity != nil {
+		in, out := &in.Affinity, &out.Affinity
+		*out = new(v1.Affinity)
+		(*in).DeepCopyInto(*out)
+	}
+	in.Resources.DeepCopyInto(&out.Resources)
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new PodPolicy.
+func (in *PodPolicy) DeepCopy() *PodPolicy {
+	if in == nil {
```

Comment:
Do you need `in.Affinity` here also as in Line 187?

Reply:
This file is generated, might be because I don’t reference Affinity outside of the type struct